CODEOWNERS

```diff
@@ -263,5 +263,5 @@
 /specification/**/resource-manager/**/readme.typescript.md @qiaozha
 /specification/**/resource-manager/**/readme.az.md @jsntcy @qiaozha
 /specification/**/resource-manager/**/readme.cli.md @jsntcy @qiaozha
-/specification/**/resource-manager/**/readme.go.md @ArcturusZhang
-/specification/**/resource-manager/**/readme.python.md @msyyc @Wzb123456789
+/specification/**/resource-manager/**/readme.go.md @tadelesh
+/specification/**/resource-manager/**/readme.python.md @msyyc
```

Comment:
Do you know if dotnet and java need the review on corresponding readme?

Reply:
DotNet does not need to review readme file. All dotnet related information is in the autorest.md in azure-sdk-for-net repo
I do not know Java.

Reply:
@weidongxu-microsoft if this is the case do we really need a readme.java.md file at all?

Reply:
@weshaggard
Java still have a few things to configure, e.g. https://github.com/Azure/azure-rest-api-specs/blob/main/specification/managednetworkfabric/resource-manager/readme.java.md

Maybe a few more directives, if swagger having problem (model with duplicate name). So it may get a bit longer than the sample above.

I am fine about either having readme.java.md, or put these java configure to readme.md. At present, we generally follow the convention of putting java specific configure in readme.java.md

What I mean is that 99% of the case, the meaningful change to readme.java.md is done by SDK dev.